target/linux/ath79/image/generic.mk

```diff
@@ -697,6 +697,16 @@ define Device/pisen_wmm003n
 endef
 TARGET_DEVICES += pisen_wmm003n
 
+define Device/pisen_ts-d084
+  $(Device/tplink-8mlzma)
```

Comment:
This comes from common-tp-link.mk ?
To be tidy, one would need to add that include in generic.mk (also missing for the other PISEN).
Practically, it is pulled in in the Makefile via generic-tp-link.mk anyhow.
Does make tolerate multiple includes of the same file? If yes, this should be added here.